packages/sw-runtime-caching/src/lib/cache-first.js

```diff
@@ -18,7 +18,18 @@ import assert from '../../../../lib/assert';
 
 /**
  * @memberof module:sw-runtime-caching
- * @extends module:sw-runtime-caching.Handler
+ * @extends Handler
```

Comment:
gotcha. Looks like gulp documentation is replacing the version of docs I had up. But I believe that has linked up the extending classes correctly. May be a jsdocs feature.